src/Page/TodoAppPage.jsx

```diff
@@ -0,0 +1,182 @@
+import React,{useState,useEffect} from 'react'
+import { v4 as uuidv4 } from "uuid";
+import './TodoAppPage.css'
+
+function TodoAppPage() {
+   const updatedTodoList=JSON.parse(localStorage.getItem("todoKeys")) || [];
```

Comment:
storage in a state not directly ,
it will fetch localStorage on every render and state change even if state change does not have relation with this variable